### DISCUSSION TITLE
Can I change background_blur with a key binding?

### DISCUSSION THREAD

**Discussion by eulersson** [kovidgoyal/kitty#7422] [Q&A] [answered]
Is it possible to change the background blur with a key binding?

I am aware it's possible to do so with background_opacity because there's a kitty action for it https://sw.kovidgoyal.net/kitty/actions/#action-set_background_opacity

Is it possible to write a kitten .py file to do it, maybe?

If it can't be done hot, I wouldn't mind having to reload the config or window.

**Comment by kovidgoyal** [comment, marked as answer]
map f1 remote_control load-config -o background_blur=whatever
